docs/application/dotnet/guides/nui/resources.md

```diff
@@ -3,14 +3,47 @@
 NUI provides several ways to handle resource images.
 This tutorial describes the use of Image resources in NUI.
```

Comment:
Image is capital here and in other places it is in lower case.

Reply:
@annie-samsung this part not in changed part, that's why there is inconsistency. But, yes this I should be smail letter i.

@mihashco request you to kindly change to:

This tutorial describes the use of image resources in NUI.